lib/appsignal/cli/install.rb

```diff
@@ -162,6 +164,24 @@ def install_for_grape(config)
           done_notice
         end
 
+        def install_for_hanami(config)
+          puts "Installing for Hanami"
+          config[:name] = required_input("  Enter application name: ")
+          puts
+          configure(config, %w[development production staging], true)
+
+          puts "Finish Hanami installation"
+          puts "  Hanami requires some manual configuration."
+          puts "  After installing the gem, add the following line to config.ru:"
```

Comment:
Should the instructions say where to add the require line? Like, "after `require hanami/boot`" or "before `Hanami.run`"? Or does it not matter?